index.js

```diff
@@ -28,6 +32,13 @@ const PREFERRED_WEIGHT = 100;
 const ANNOTATIONS_PATH = 'metadata.annotations';
 const CONTAINER_WAITING_REASON_PATH = 'status.containerStatuses.0.state.waiting.reason';
 
+const DOCKER_ANNOTATIONS = [
+    'screwdriver.cd/dockerenabled',
```

Comment:
we've been using camel case
https://docs.screwdriver.cd/user-guide/configuration/annotations